lib/soft_delete.rb

```diff
@@ -4,7 +4,7 @@
 require 'soft_delete/version'
 require 'soft_delete/soft_deletable'
 require 'soft_delete/restorable'
-require_relative 'soft_delete/configuration'
+require 'soft_delete/configuration'
```

Comment:
This change is not making any difference as such, just making uniform with other require statments.